native/packages/react-native-bpk-component-button-link/src/BpkButtonLink.android.js

```diff
@@ -40,6 +40,7 @@ export type Props = {
   ...$Exact<CommonProps>,
   borderlessBackground: boolean,
   style: ?any,
+  allowFontScaling: boolean,
```

Comment:
Actually, i should have thought about this further. Rather than exposing individual props to the underlying `BpkText`, we should expose a `textProps` prop that we spread on to the component:

```
<BpkButtonLink
  title={doneLabel}
  onPress={onClose}
  textProps={{
    allowFontScaling={false}
  }}
/>
```

This solves the use case and prevents us from having to make changes going forward. I.e. there are [lots more props ](https://facebook.github.io/react-native/docs/text.html#props)that we might want to pass.